### PR TITLE
SDA-6528: Add Hypershift endpoint

### PR DESCRIPTION
**Pull request by andreadecorte**
Add an endpoint to get Hypershift cluster info
Align case usage with CS to use Hypershift instead of HyperShift

### CODE REVIEW

**Comment by igoihman** [comment]
@tbrisker @etabak PTAL

**Comment by tbrisker** [comment]
@andreadecorte thanks! looks like this change hasn't been made to CS yet, so we cannot merge the api model update until it's implemented. Converting to draft for now so it doesn't get merged by mistake, feel free to switch it back once the backend has been updated.

**Comment by andreadecorte** [comment]
> @andreadecorte thanks! looks like this change hasn't been made to CS yet, so we cannot merge the api model update until it's implemented. Converting to draft for now so it doesn't get merged by mistake, feel free to switch it back once the backend has been updated.

makes sense, thanks

File: model/clusters_mgmt/v1/hypershift_resource.model [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
```
[the remark this answers is not in the document]
Comment (andreadecorte) [reply model/clusters_mgmt/v1/hypershift_resource.model]: fixed, thanks

**Comment by andreadecorte** [comment]
updated PR to separate public and private Hypershift config

**Comment by andreadecorte** [comment]
CS code has been merged and loaded in production, this should be good to merge now

**Review by tbrisker — APPROVED** [review]

**Pull request merged**